common/app/implicits/Requests.scala

```diff
@@ -24,7 +24,7 @@ trait Requests {
 
     lazy val isAmp: Boolean = r.getQueryString("amp").isDefined || (!r.host.isEmpty && r.host == Configuration.amp.host)
 
-    lazy val isEmail: Boolean = (r.getQueryString("format") == Some("email")) || r.path.endsWith(EMAIL_SUFFIX)
+    lazy val isEmail: Boolean = r.getQueryString("format").contains("email") || r.path.endsWith(EMAIL_SUFFIX)
```

Comment:
personally I'd rather have check that the url is exactly format=email or empty, and return a 404 if it has any other parameter exists or other value.  That means that we can't ever be unknowningly caching the same content for format=blah, format=fish, format=email&CMP=fb_gu, and standard.